tests/v2/test_waku_rln_relay.nim

```diff
@@ -600,13 +600,13 @@ suite "Waku rln relay":
       var memberAdded: bool = false
       if (i == index):
         # insert the current peer's pk
-        memberAdded = rln.insertMember(memKeys.idCommitment)
+        memberAdded = rln.insertMembers(uint(i), @[memKeys.idCommitment])
```

Comment:
As an idea, if you want to avoid having 

since i see 7 `uint()` conversions, perhaps you want to use this? (or whatever uintxx you need)

```nim
for i in 0'u..10'u:
  echo i
```

Reply:
Addressed in d40636d0